radar/infrastructure_config/docker_swarm.yml

```diff
@@ -0,0 +1,15 @@
+version: 1
+name: Docker Swarm
+blip:
+  - version: "2.6"
+    ring: TRIAL
```

Comment:
Is this used anywhere? Otherwise probably `ASSESS`. Are we actively looking for alternatives to Kubernetes?

Reply:
Changing to ASSESS